dev-libs/libsecp256k1/libsecp256k1-0.5.1.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+MyPN=secp256k1
+DESCRIPTION="Optimized C library for EC operations on curve secp256k1"
+HOMEPAGE="https://github.com/bitcoin-core/secp256k1"
+SRC_URI="https://github.com/bitcoin-core/secp256k1/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0/2"  # subslot is "$((_LIB_VERSION_CURRENT-_LIB_VERSION_AGE))" from configure.ac
+KEYWORDS="~amd64 ~arm ~arm64 ~mips ~ppc ~ppc64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="+asm +ecdh +ellswift experimental +extrakeys lowmem +recovery +schnorr test valgrind"
+RESTRICT="!test? ( test )"
+
+REQUIRED_USE="
+	asm? ( || ( amd64 arm ) arm? ( experimental ) )
+	schnorr? ( extrakeys )
+"
+BDEPEND="
+	dev-build/autoconf-archive
+	virtual/pkgconfig
+	valgrind? ( dev-debug/valgrind )
+"
+
+PATCHES=(
+	"${FILESDIR}/0.4.0-fix-cross-compile.patch"
+)
+
+S="${WORKDIR}/${MyPN}-${PV}"
```

Comment:
You seemed to have move this from its place under `SRC_URI`, if you have a copy in your overlay please update it to match the one in the tree after 0a517b74a98df1af31142d9eaa5053f0a2ef5648.

Reply:
I did copy 0.5.0 to 0.5.1 before I had reset my worktree to match the upstream HEAD. Sorry about that.

Reply:
No worries, I just wanted to notify you in case you had an overlay somewhere else where you kept copying these ebuilds :)